CoreFindView/src/au/gov/asd/tac/constellation/views/find2/FindViewController.java

```diff
@@ -107,10 +109,10 @@ public FindViewController init(final FindViewTopComponent parentComponent) {
             try {
                 future.get();
             } catch (final InterruptedException ex) {
-                Exceptions.printStackTrace(ex);
+                LOGGER.log(Level.SEVERE, ex.getMessage());
```

Comment:
These ones where you've replaced the stack trace print with a log (similar in other classes) are you able to add the exception as a parameter?
(e.g. `LOGGER.log(Level.SEVERE, ex.getMessage(), ex)`)